modulefiles/OznMonBuild.wcoss2

```diff
@@ -0,0 +1,18 @@
+#%Module################################################
```

Comment:
Is this used in the build?
I thought @EdwardSafford-NOAA mentioned that gfsv16.1 does not use this modulefile, but instead uses the one from GSI.

Reply:
Correct, it's not used per @EdwardSafford-NOAA's confirmation, so I removed the RadMon/OznMonBuild.wcoss2 I had initially added. Will use what comes from the GSI instead. I can retire the other platform copies of the RadMon/OznMonBuild files in the develop branch separately.